package.json

```diff
@@ -19,7 +19,8 @@
     "pretest": "npm run build",
     "test": "mocha \"test/**/*-test.coffee\"",
     "test:coverage": "scripts/coverage.sh",
-    "prepublish": "npm run build",
+    "prepare": "npm run build",
+    "prepublish": "check-node-version --npm \">=4\" || npm run prepare",
```

Comment:
Ah, I did not notice the commit message during review:

> See https://iamakulov.com/notes/npm-4-prepublish/ , npm/npm#16685 , and https://stackoverflow.com/a/44136814/1098906

Could you send this change in a separate PR, please? At the same time, I'm still not completely sure about it, because Dredd is being published on CI exclusively, so we could just make sure that `package.json` contents match the npm version which publishes the package. That basically means just pinning npm to a specific version in the Travis CI config and then having the `package.json` specific to that npm version.